src/features/home-calendar/ui/style.css.ts

```diff
@@ -0,0 +1,91 @@
+import { style } from '@vanilla-extract/css';
+import { font } from 'shared/styles/font.css';
+import theme from 'shared/styles/theme.css';
+
+export const MiniCalendarContainer = style({
+  padding: '12px',
+  borderRadius: '12px',
+  flexShrink: 0,
+  alignSelf: 'stretch',
+  backgroundColor: theme.white,
+  overflow: 'hidden',
+});
+
+export const MiniCalendarHeader = style({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+});
+
+export const MiniCalenadrTitle = style({
+  color: theme.black,
+  textAlign: 'center',
+  fontFamily: 'Pretendard',
+  fontSize: font.H5.fontSize,
+  fontWeight: '500',
+  lineHeight: '160%',
+  fontStyle: 'normal',
+});
+
+export const MiniCalendarGrid = style({
+  display: 'grid',
+  gridTemplateColumns: 'repeat(7, 1fr)',
+  rowGap: '8px',
+  justifyContent: 'space-between',
+  margin: '8px 4px 0 4px',
+  justifyItems: 'center',
+});
+
+export const MiniCalenadrWeek = style({
```

Comment:
Calenadr 오타인거 같은데 수정부탁드립니다.
```suggestion
          className={s.MiniCalendarWeek}
```